Documentation/GettingStarted.md

```diff
@@ -54,9 +57,112 @@ Add a user to the libary. The user can be of any class, as long the calls implem
         NSLog(@"Failed to create Account: %@", error);
     }
 ```
+### 4. Inbound calls
 
-#### 4. Set callback for incoming calls
+#### 1. AppDelegate - instantiate CallKitProviderDelegate to a private property 
 
+Objective-C
 ```objective-c
-TODO
+    - (CallKitProviderDelegate *)callKitProviderDelegate {
+        if !(_callKitProviderDelegate) { 
+            _callKitProviderDelegate = [[CallKitProviderDelegate alloc] initWithCallManager:[[VialerSIPLib sharedInstance].callManager]]; 
+        }
+        return _callKitProviderDelegate;
+    }
+```
+
+Swift
+```swift
+    var providerDelegate: CallKitProviderDelegate?
+    providerDelegate = CallKitProviderDelegate(callManager: VialerSIPLib.sharedInstance().callManager)
+```
+
+#### 2. Adapt the incoming call callback block.
+
+Objective-C
+```objective-c
+    if ([VialerSIPLib callKitAvailable]) { 
+        [self.callKitProviderDelegate reportIncomingCall:call]; 
+    } else { 
+        // Your current inbound call handling	
+    }
+```
+
+#### 3. The CallKit provider delegate informs your app about incoming calls through 2 notifications:
+
+- CallKitProviderDelegateOutboundCallStarted
+- CallKitProviderDelegateInboundCallAccepted
+
+These notification should be used by your app to display the appropriate calling screen.
+When receving this notification, a VSLCall object is sent with it an can by found in the notification's
+user info dict with the key: VSLNotificationUserInfoCallKey.
+All operations on a VSLCall should now be done through the new VSLCallManager class. This means:
+- starting a call to a number
+- answering an inbound call
+- ending a call
+- toggle mute and hold
+- sending DTMF tones
+
+You can also query the CallManger for a call based on "UUID" or "PJSIP call ID".
+
+### 5. Outbound calls
+
+Use the new VSLCallManager -startCall function to instantiate an outbound call.
+
+To be able to use the app through a native iOS interface, e.g. recents or contacts, you will need to create an "Intent Extension".
```

Comment:
I don't think this is because of Siri, there is another solution for that in SiriKit with intents.
This is when you want to have it integrated in your Addressbook & recents call list. Don't know if you leave out this part the calls won't show up in your recents (but I guess so).